libraries/bezier-rs/src/subpath/solvers.rs

```diff
@@ -58,6 +58,64 @@ impl Subpath {
 
 		intersection_t_values
 	}
+
+	/// Return the min and max corners that represent the bounding box of the subpath.
+	pub fn bounding_box(&self) -> Option<[DVec2; 2]> {
+		let bounding_boxes = self.iter().map(|bezier| bezier.bounding_box()).collect::<Vec<[DVec2; 2]>>();
+		if bounding_boxes.is_empty() {
+			// There is no bounding box for empty subpath.
+			None
+		} else {
+			let mut min_points = bounding_boxes[0][0];
+			let mut max_points = bounding_boxes[0][1];
+			for bounding_box in bounding_boxes {
+				// We take min of mins and max of maxes.
+				min_points = min_points.min(bounding_box[0]);
+				max_points = max_points.max(bounding_box[1]);
+			}
+			Some([min_points, max_points])
+		}
+	}
+
+	/// Returns two lists of `t`-values representing the local extrema of the `x` and `y` parametric subpaths respectively.
+	/// The list of `t`-values returned are filtered such that they fall within the range `[0, 1]`.
+	pub fn local_extrema(&self) -> [Vec<f64>; 2] {
+		let (mut x_extremas, mut y_extremas) = (Vec::new(), Vec::new());
+		let number_of_curves = self.len_segments() as f64;
+
+		for (index, bezier) in self.iter().enumerate() {
+			// Convert t-values of bezier curve to t-values of subpath
+			let extremas = bezier.local_extrema();
+			x_extremas.extend(extremas[0].iter().map(|t| ((index as f64) + t) / number_of_curves)
+					.collect::<Vec<f64>>());
+			y_extremas.extend(extremas[1].iter().map(|t| ((index as f64) + t) / number_of_curves)
+					.collect::<Vec<f64>>());
+		}
+
+		// TODO: Consider the point between bezier curves.
```

Comment:
This might be more clear if you change the wording to: "Consider the shared point between adjacent beziers"